tests/resources/examples_erroneous/errors/results.csv

```diff
@@ -21,3 +22,7 @@
 "observation.csv","Observation","Invalid timestamp format. Expecting ""YYYY-MM-DD HH:MM:SS[.SSSSSS]"": line number 3","observation_datetime","04-15-1987 06:06:30.6260548","timestamp"
 "observation.csv","Observation","Type mismatch line number 3","observation_type_concept_id","unknown","integer"
 "visit_detail.csv","Visit Detail","NULL values are not allowed for column","visit_occurrence_id","",""
+"note.jsonl","Note","Column not in table definition","favorite_note_id","favorite_note_id",""
+"note.jsonl","Note","Column missing in file","note_type_concept_id","","note_type_concept_id"
+"note.jsonl","Note","Column not in expected order","favorite_note_id","favorite_note_id","note_type_concept_id"
+"note.jsonl","Note","Missing required column","note_type_concept_id","",""
```

Comment:
It prints the full stacktrace here for some json decode errors, but I think that's fine. It is less readable but has more info. A different stacktrace gets printed in the console, but without the erroneous line number and as a ValueError. Can you print the erroneous line number there as well?